test/app_test.dart

```diff
@@ -74,15 +74,15 @@ Future<void> main([List<String>? args]) async {
 
   baasTest('App log out no current user is no operation and does not crash', (configuration) async {
     final app = App(configuration);
-    await app.logout();
+    await app.currentUser?.logOut();
   });
 
   baasTest('App log out user', (configuration) async {
     final app = App(configuration);
     final user = await app.logIn(Credentials.emailPassword(testUsername, testPassword));
 
     expect(user.state, UserState.loggedIn);
-    await app.logout(user);
+    await user.logOut();
```

Comment:
We are testing the app in app_test.dart file. Isn't it better here to use `app.logout(user)`? In user_test.dart you have already tests for `user.logOut()`.